### ISSUE TITLE
License

### ISSUE THREAD

**Issue by ternaus** [cvqluu/Additive-Margin-Softmax-Loss-Pytorch#9]
Hi. Could you please add a license file?
